### ISSUE TITLE
html.elements.s - Safari with VoiceOver does not announce text preceding `<s>` element

### ISSUE THREAD

**Issue by aardrian** [mdn/browser-compat-data#27446]
### What type of issue is this?

Incorrect support data (example: BrowserX says "86" but support was added in "40")

### What information was incorrect, unhelpful, or incomplete?

Support for the `<s>` element is incomplete / has regressed in Safari 18 (through 18.5 as of this writing). When paired with VoiceOver it functionally removes preceding content from the page.

### What did you expect to see?

When using VoiceOver with Safari, that the text preceding a `<s>` would be announced.'

### Did you test this? If so, how?

In a test using `<s>` or `role="deletion"`, VoiceOver with Safari does not announce the opening of the sentence, instead jumping to where the element starts.

To replicate:

1. Open Safari and activate VoiceOver
2. Go to:
   * https://cdpn.io/aardrian/debug/rNRxZmP
   * https://adrianroselli.com/demos/text-level-semantics-reprised/
3. Use the `read-all` command (VO + A)
4. Observe how, for most paragraphs, the words preceding the element are not announced.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

[286267: AX: VoiceOver does not announce text preceding some phrasing elements](https://bugs.webkit.org/show_bug.cgi?id=286267)

### Do you have anything more you want to share?

https://github.com/user-attachments/assets/796d1401-a47c-4cde-ba30-4217ecfc50c6

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/s

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `html.elements.s`
* Report started: 2025-07-29T21:51:53.160Z

</details>

**Comment by aardrian** [comment]
Related:
* https://github.com/mdn/browser-compat-data/issues/27443
* https://github.com/mdn/browser-compat-data/issues/27444
* https://github.com/mdn/browser-compat-data/issues/27445
* https://github.com/mdn/browser-compat-data/issues/27446

**Comment by lolaodelola** [comment]
Hi @aardrian, do you know if this issue is present when using VO with Firefox or Chrome?

**Comment by aardrian** [comment]
@lolaodelola 
> do you know if this issue is present when using VO with Firefox or Chrome?

It is not. But most VO users aren't using those browsers owing to known challenges with their access to the macOS AAPIs.

**Comment by lolaodelola** [comment]
@aardrian thanks, noted!

**Comment by caugner** [comment]
Closing as won't fix, see https://github.com/mdn/browser-compat-data/issues/27443#issuecomment-3179926464.
